src/Admin/Pool.php

```diff
@@ -41,9 +42,9 @@ class Pool
     protected $adminClasses = [];
 
     /**
-     * @var string[]
+     * @var MutableTemplateRegistryInterface
      */
-    protected $templates = [];
+    protected $templateRegistry;
```

Comment:
Should be private

Reply:
Should I add a `final protected getTemplateRegistry()` so people can access it in their admins?

Reply:
No, if they need to, they will ask for it.

Reply:
Plus this is `Pool`, so that wouldn't help, I guess, since `protected` would not be enough

Reply:
Oh, you're right.